packages/jest-cli/src/cli/args.ts

```diff
@@ -349,7 +350,7 @@ export const options = {
       'will spawn for running tests. This defaults to the number of the ' +
```

Comment:
While on it, maybe we should also update this description to mention that
* it can also be specified as a percentage of cores
* it defaults to the number of the cores available on your machine *minus one*

If the description right above says this it's also likely a review would catch the error if it's changed to number again :)

Reply:
Right, I think that’s a great idea. Will update that.

I also stumbled upon the case that the config file jest.config.js currently also only allows numbers and not % input. But that one is not documented as existing anyways. Could be another easy to start on pull request.

Reply:
would be even cooler to fix validation in this PR straight away :)

Reply:
Would it though ?? :D 

Ok fine I will do it, but this will probably have to wait till the weekend.

Reply:
Ok done.